ntbs-service/Pages/Notifications/OverviewPartials/_TreatmentEventsOverviewPartial.cshtml

```diff
@@ -51,7 +51,7 @@
                     @(period.IsTransfer ? "Transfer" :  $"Treatment period {period.PeriodNumber}")
                 </h4>
 
-                @if (period.TreatmentEvents.Last().IsEpisodeEndingTreatmentEvent())
+                @if (period.TreatmentEvents.Last().TreatmentEventType == TreatmentEventType.Denotification || period.TreatmentEvents.Last().IsEpisodeEndingTreatmentEvent())
```

Comment:
Wouldn't it be better to change the definition of `IsEpisodeEndingTreatmentEvent` here?

Reply:
Yes I've changed this now.